docs/tasks/administer-cluster/distribute-credentials-secure.md

```diff
@@ -0,0 +1,122 @@
+---
+---
+
+{% capture overview %}
+This page shows how to create a Secret and a Pod that has access to the Secret.
```

Comment:
It might be worth having a separate bulleted section of "related concepts" (Secret, Pod).  WDYT?